flent/scripts/ss_iterate.sh

```diff
@@ -41,7 +41,8 @@ then
 fi
 
 command_string=$(cat <<EOF
-for i in \$(seq $count); do
+endtime=\$(date -d "\$length sec" +%s%N);
```

Comment:
Here you shouldn't be escaping the `$` - we want it evaluated while setting the command, not on the remote host.

Reply:
Oh, But wouldn't it be better if the endtime was calculated in the remote host (For e.g if ssh took few seconds to connect, then the stats collection time will be reduced) or am I missing something?

Reply:
Ah, yes, the sorry, the first `$` should be escaped, but not the second one. I.e., `$length` should be evaluated straight away...